### ISSUE TITLE
Input selector for Onkyo TR-NR609

### ISSUE THREAD

**Issue by padman74** [ToddGreenfield/homebridge-onkyo#4]
Hi Todd, 
thank you so much for your work, much appreciated. 
One issue that I faced today. When I try to use the input selector in EVE app the plugin says "Command input-selector in zone main is not supported on this model". Can you please confirm that this is true. I thought I read that you tested this with TR-NR609. Is there any workaround to switch by scene to another input?

Regards
Volker

**Comment by ToddGreenfield** [comment]
Hi. I actually have an TX-8050. The docs show the NR609 as a holdover from the original author when it was hard coded. So I am unable to test with that model. Is that your real model? If not, what is your real model number?
Are you able to switch inputs from the Onkyo Remote App?

**Comment by padman74** [comment]
Hi again. Yes, model number is TX-NR609 and all remote apps that I‘ve tested work without any problems.

**Comment by padman74** [comment]
Even the default input channel that we can configure in the homebridge config file while starting is understood by my receiver.

**Comment by ToddGreenfield** [comment]
Now that I had a chance to look at the code again, this is normal/expected. Due to the way I had to implement this functionality, if using the slider in EVE to change/set the input and if set to an invalid input (something that doesn't map to or is available for a given receiver), whether unintentionally or not, the EVE app should readout "SET ERROR" and the logfile would show the error you noted above. Again, this is expected as I didn't know of a way to provide input other than a slider.

It may be easiest to set the volume or Input first using the OnkyoRemote3 app and then creating the scenes so the volume or input is pre-set (without using the slider). That is how I set inputs. Let me know how that works for you.
